edge-modules/azure-monitor/src/buildscript.bash

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Remove this script, looks like we don't need it.

Reply:
Removed.